crates/rethnet_evm/benches/state_history.rs

```diff
@@ -0,0 +1,141 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+
+use rethnet_eth::B256;
+
+mod util;
+use util::{bench_sync_state_method, prep_no_op, SNAPSHOT_SCALES, STORAGE_SCALES};
+
+fn bench_checkpoint(c: &mut Criterion) {
+    bench_sync_state_method(
+        c,
+        "StateHistory:checkpoint()",
+        prep_no_op,
+        |mut state, _number_of_accounts, _, _| {
+            let result = state.checkpoint();
+            debug_assert!(result.is_ok());
```

Comment:
Yes, our CI and git pre-commit hook are both configured to run `cargo test --workspace --all-targets --features tracing,bench-once`. That command, due I believe to `--all-targets`, also runs the benchmarks as a test, and the `bench-once` bit ensures that we just run through a single permutation of the benchmarks, so that it can finish quickly.